wandb/sdk/launch/runner/abstract.py

```diff
@@ -61,6 +64,11 @@ def __init__(self) -> None:
     def status(self) -> Status:
         return self._status
 
+    @abstractmethod
+    def get_logs(self) -> Optional[str]:
+        """Returns the logs associated with the run."""
```

Comment:
nit: imperative comments (i think ruff cares?)
```suggestion
        """Return the logs associated with the run."""
```